client/src/components/AuthenticationForm.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Input from './Input';
+import Button from './Button';
+import styled from '@emotion/styled';
+import colors from '../utils/colorsUtils';
+import PropTypes from 'prop-types';
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  > * {
+    flex-basis: 45%;
+  }
+`;
+
+const InputContainer = styled.div`
+  border: solid 2px ${colors.primaryYellow};
```

Comment:
Not related to this pull request:
Imagine that you have two themes. A colorful and grayscale theme. 
`primaryYellow` already tells you how the color looks like. This makes it not possible to have a different border color.
Theme color is more generic and doesn't tell you the value of the color.

Maybe call it `colors.border`?